src/python/CRABClient/JobType/UserTarball.py

```diff
@@ -9,6 +9,8 @@
 import tarfile
 import tempfile
 
+from  __future__ import division
```

Comment:
Very minor, but could you please move this above the other imports? Below the comment block, that's where we usually put these future imports.